lib/ex_unit/lib/ex_unit.ex

```diff
@@ -263,4 +263,23 @@ defmodule ExUnit do
   def run do
     ExUnit.Runner.run(configuration(), nil)
   end
+
+  # Configures on demand defaults
+  defp configure_defaults(options) do
+    unless Keyword.has_key?(options, :seed) do
+      Application.put_env(:ex_unit, :seed, :os.timestamp |> elem(2))
+    end
+
+    unless Keyword.has_key?(options, :max_cases) do
```

Comment:
This is a change in behaviour. `options[:trace]` sets the number of cases to 1 even if `:max_cases` is set. My suggestion would be to configure_defaults BEFORE calling `configure` and have it set the defaults in the `options` keywords instead of directly in the app environment.